test_conformance/api/test_api_min_max.cpp

```diff
@@ -1946,64 +1946,90 @@ int test_min_max_device_version(cl_device_id deviceID, cl_context context, cl_co
         return -1;
     }
 
-    // Make sure 2.0 devices support required extensions for 2.0
-    const char *requiredExtensions20[] =
-    {
-        "cl_khr_byte_addressable_store",
+    // Make sure 2.x devices support required extensions for 2.x
+    // note: these extensions are **not** required for devices
+    // supporting OpenCL-3.0
+    const char *requiredExtensions2x[] = {
         "cl_khr_3d_image_writes",
         "cl_khr_image2d_from_buffer",
         "cl_khr_depth_images",
-        "cl_khr_fp64",
     };
 
     // Make sure 1.1 devices support required extensions for 1.1
-    const char *requiredExtensions[] =
-    {
+    const char *requiredExtensions11[] = {
         "cl_khr_global_int32_base_atomics",
         "cl_khr_global_int32_extended_atomics",
         "cl_khr_local_int32_base_atomics",
         "cl_khr_local_int32_extended_atomics",
         "cl_khr_byte_addressable_store",
-        NULL
     };
 
-    if( major * 10 + minor >= 11 )
+
+    int version_as_integer = major * 10 + minor;
+    if (version_as_integer >= 11)
     {
         log_info( "Checking for required extensions for OpenCL 1.1 and later devices...\n" );
-        for( i = 0; NULL != requiredExtensions[i]; i++ )
+        for (i = 0;
+             i < sizeof(requiredExtensions11) / sizeof(requiredExtensions11[0]);
```

Comment:
Please use `ARRAY_SIZE`

Reply:
Thanks for your feedback, I have updated to use `ARRAY_SIZE`.